docs/using_plugins.rst

```diff
@@ -279,17 +279,21 @@ method. Continuing from the example above:
 
 .. code-block:: python
 
-    def my_custom_field2properties(field, **kwargs):
+    def my_custom_field2properties(self, field, **kwargs):
```

Comment:
Maybe rename the first arg to make it more clear? `converter`?

Reply:
Yeah, I was wondering.

The point is that the custom and stock functions look alike. And the doc says the function will be bound. We could stress out bound -> self, or s/self/converter. But converter is not that self-explanatory either.

No strong feeling either way. In any case, it's a trick in trick's clothing.

Reply:
Fair enough. I've no strong opinion either way. `self` just looks strange on a function, but yeah..'trick's clothing'. =)